frontend/apps/web/components/ui/form.tsx

```diff
@@ -153,14 +154,17 @@ const FormMessage = React.forwardRef<
   }
 
   return (
-    <p
-      ref={ref}
-      id={formMessageId}
-      className={cn('text-[0.8rem] font-medium text-destructive', className)}
-      {...props}
-    >
-      {body}
-    </p>
+    <div className="flex flex-row items-center gap-2 text-red-900 dark:text-red-100 border border-red-700 bg-red-100 dark:bg-red-950 rounded-xl p-1 px-2">
+      <MdErrorOutline />
+      <p
+        ref={ref}
+        id={formMessageId}
+        className={cn('text-xs font-medium ', className)}
+        {...props}
+      >
+        {body}
+      </p>
+    </div>
```

Comment:
Hey this is a shadcn component and generally shouldn't be touching stuff in here in an effort to keep it pretty upstream with shadcn so that it makes it easy to pull  in future updates.

In other words, why the change to this now? And does this styling look okay throughout the rest of the app for all of our forms that use FormMessage ?

Reply:
yeah i updated it since it looked different compared to the other warning that we show in the connect page now. I checked in other places and I (personally) think it looks much better.

Reply:
yeah i updated it since it looked different compared to the other warning that we show in the connect page now. I checked in other places and I (personally) think it looks much better. But good catch, I only want it to be applied to error cases so will add a check here for that.